generators/ionic/resources/base/cypress.json

```diff
@@ -0,0 +1,12 @@
+{
+  "baseUrl": "http://localhost:8100/",
```

Comment:
This doesn't seem like it's being used. I think it'd be awesome if e2e tests could run on 8100 since that was the previous behavior. It also makes it so you don't have to add two callback URIs.

Reply:
Configured `ng serve` to use port 8100, so no 4200 port is not needed anymore.